test/e2e/manifests/apiextensions/composition/circuit-breaker/xr.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: example.org/v1alpha1
+kind: CircuitBreakerTest
+metadata:
+  name: circuit-breaker-xr
+spec:
+  message: "Testing circuit breaker functionality"
+  crossplane:
+    compositionRef:
+      name: circuit-breaker-composition
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)